app/routers/calendar_grid.py

```diff
@@ -0,0 +1,47 @@
+import calendar
+import itertools
+from datetime import datetime, timedelta
+
+from typing import Any, Iterator, Generator, List, Tuple
+
+DAYS_OF_THE_WEEK: List[str] = ['Monday', 'Tuesday', 'Wednesday',
```

Comment:
I'd like to believe it's part of some package we will use ?

Reply:
I've commented about this on the previous submit. The weekdays should not be hardcoded to English here. The calendar module has built in support for the the weekdays and also has support for weeks in different order. That should be used instead of this.

Reply:
There is an open ticket about this. I guess it will be take care soon.